app/src/main/java/org/systers/mentorship/view/fragments/RelationFragment.kt

```diff
@@ -104,12 +106,22 @@ class RelationFragment : BaseFragment() {
             tvEndDate.text = convertUnixTimestampIntoStr(
                     relationResponse.endsOn, EXTENDED_DATE_FORMAT)
             tvRelationNotes.text = relationResponse.notes
-
             btnCancelRelation.visibility = View.VISIBLE
             btnCancelRelation.setOnClickListener {
-                relationViewModel.cancelMentorshipRelation(relationResponse.id)
+
+                val cancelDialog = LayoutInflater.from(activity).inflate(R.layout.dialog_cancel_relation, null,true)
+
+                with(alertDialog) {
+                    this?.setView(cancelDialog)
```

Comment:
try to create the dialog like in PR #141 , it seems it has less custom configurations, that you don't need here.
With this way of doing things, you won't need the custom layout for the dialog.